packages/bsky/tests/views/feed-view-post.test.ts

```diff
@@ -0,0 +1,336 @@
+import { AtpAgent, AppBskyFeedDefs, AtUri } from '@atproto/api'
+import { TestNetwork, SeedClient, basicSeed } from '@atproto/dev-env'
+
+/**
+ * To ensure consistency, we only test based on the data returend from a single
+ * `FeedViewPost`, since the frontend constructs feed slices based on a single
+ * `FeedViewPost`.
+ *
+ * With more results in the first page of data, other computations are
+ * feasible, but since they rely on when they were created, results are
+ * inconsistent.
+ */
+const LIMIT = 1
+
+describe('pds thread views', () => {
+  let network: TestNetwork
+  let agent: AtpAgent
+  let pdsAgent: AtpAgent
+  let sc: SeedClient
+
+  // account dids, for convenience
+  let alice: string
+  let bob: string
+  let carol: string
+  let dan: string
+
+  beforeAll(async () => {
+    network = await TestNetwork.create({
+      dbPostgresSchema: 'bsky_views_feed_thread_slices',
```

Comment:
Looks like the name of the test file, the `describe()` for the test suite, and the db postgres schema may need to get aligned with each other.